<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ANUSRI" failures="0" tests="3" name="DetailsOfAmazon.Website.TestCase10_UpdatingandRemovingfromCart" time="105.624" errors="1" timestamp="2025-07-15T12:48:05 IST" skipped="2">
  <testcase classname="DetailsOfAmazon.Website.TestCase10_UpdatingandRemovingfromCart" name="UpdatingCart" time="35.202">
    <skipped/>
  </testcase> <!-- UpdatingCart -->
  <testcase classname="DetailsOfAmazon.Website.TestCase10_UpdatingandRemovingfromCart" name="UpdatingCart" time="34.604">
    <error message="Cannot invoke &quot;org.openqa.selenium.WebDriver.getClass()&quot; because &quot;exceptionDriver&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebDriver.getClass()" because "exceptionDriver" is null
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:85)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
at DetailsOfAmazon.Website.CartPage.AddtoCart1(CartPage.java:34)
at DetailsOfAmazon.Website.TestCase10_UpdatingandRemovingfromCart.UpdatingCart(TestCase10_UpdatingandRemovingfromCart.java:24)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
    <system-out>
      <![CDATA[Testcase is failed]]>
    </system-out>
  </testcase> <!-- UpdatingCart -->
  <testcase classname="DetailsOfAmazon.Website.TestCase10_UpdatingandRemovingfromCart" name="UpdatingCart" time="35.818">
    <skipped/>
  </testcase> <!-- UpdatingCart -->
  <system-out>
    <![CDATA[Testcase is passed
Testcase is passed
Testcase is passed
Testcase is failed
Testcase is passed
Testcase is passed
Testcase is passed
Testcase is passed
Testcase is failed
Testcase is passed
Testcase is passed
Testcase is passed
Testcase is passed
Testcase is passed
Testcase is passed]]>
  </system-out>
</testsuite> <!-- DetailsOfAmazon.Website.TestCase10_UpdatingandRemovingfromCart -->
